docs/04-graphics.mdx

```diff
@@ -327,9 +327,11 @@ const spriteSheet = new ex.SpriteSheet({
 })
 ```
 
-If you spritesheet is a neat grid there is a static builder for you to slice up that image source. Most sprite sheets are tightly packed like so.
+If you spritesheet is a neat grid, there is a static builder you can use to slice up that image source. Most sprite sheets are tightly packed.
 
-Some source spritesheets may have margin between sprites and an offset, like these [playing cards from Kenny.nl](https://www.kenney.nl/assets/playing-cards-pack)
+// TODO example image missing
```

Comment:
I see the example on the live site, it's being included with the "embed" snippet

Reply:
Oh nvm, the example for tightly packed is missing